scripts/map_peaks_to_known_genes.py

```diff
@@ -16,9 +19,9 @@
 """
 epilog = ''
 parser = OptionParser(usage=usage,description=description,epilog=epilog)#,formatter=MultiLineHelpFormatter())
-parser.add_option('--upstream-window',dest='upst_win',type='int',default=100000,help='window width in base pairs to consider promoter region [default: %default]')
-parser.add_option('--downstream-window',dest='dnst_win',type='int',default=0,help='window width in base pairs to consider downstream region [default: %default]')
-parser.add_option('--tss',dest='tss',action='store_true',default=False, help='calculate downstream window from transcription start site instead of transcription end site')
+parser.add_option('--upstream-window',dest='upst_win',type='int',default= 2000,help='window width in base pairs to consider promoter region [default: %default]')
```

Comment:
That's a good reason to change the defaults, thanks for explaining it to me.  I agree with the change, but let's remember to add it in our release notes.  Maybe we should keep a running log of release notes for future releases in `HISTORY.rst`?  For release 0.3.0 I read through all the old commits to compile the release notes, which took a long time.